vagrant-kubernetes-istio/README.md

```diff
@@ -67,36 +64,37 @@ make e2e_simple E2E_ARGS="--use_local_cluster"
 ```
 
 ### 3. Debug with Delve
-You can try debugging Istio with debugger tool. vm_setup.sh already installs Delve into VM for us. To use Delve, we need process id of the binary we want to debug.
-For example, if you want to debug a test, run that test in your host/vm.
-```bash
-# In the VM/Host
-make e2e_simple E2E_ARGS="--use_local_cluster --skip_cleanup"
-```
-Now, find the pid of the process say pilot-discovery that you want to debug.
+[vm_setup.sh](vm_setup.sh) installs Delve in the VM. In order to debug Istio, we need the process ID of the binary in the VM.
+For example, find the PID of the pilot-discovery process to debug.
 ```bash
 vagrant ssh
 ps -ef | grep pilot-discovery
 ```
-Then, you can run Delve
+Then, run Delve
 ```bash
 sudo -E env "PATH=$PATH" dlv attach <pid of pilot-discovery>
 ```
-For more information, please check [Debug an Istio container with Delve](https://github.com/istio/istio/wiki/Dev-Guide#debug-an-istio-container-with-delve)
+and kick of the tests from the host of VM.
```

Comment:
I am confused about "kick of the tests ...", before we run the test, pilot-discovery process does not exist.
That is why we have make e2e_simple E2E_ARGS="--use_local_cluster --skip_cleanup" at line 74, so that the process is still alive after the test, and we can run Delve to debug this process.